tests/e2e-openshift/otlp-metrics/04-metrics-traces-gen.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: otelgen-traces
+spec:
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      labels:
+        app: otelgen
+    spec:
+      containers:
+        - name: otelgen-traces-multi
+          image: ghcr.io/krzko/otelgen:latest
```

Comment:
I would prefer to use official tooling e.g. ghcr.io/open-telemetry/opentelemetry-collector-contrib/telemetrygen

Reply:
used the offical tooling.